modules/fontrendering/src/processors/textoverlaygl.cpp

```diff
@@ -59,49 +82,37 @@ TextOverlayGL::TextOverlayGL()
     , inport_("inport")
     , outport_("outport")
     , enable_("enable", "Enabled", true)
-    , text_("text", "Text", "Lorem ipsum etc.", InvalidationLevel::InvalidOutput,
-            PropertySemantics::TextEditor)
+    , texts_{"texts", "Texts", std::make_unique<TextOverlayProperty>("text0", "Text 0"), 99,
```

Comment:
why the 99 limit here?

Reply:
you had it in there before since you only parsed %99, think I have removed it now...

Reply:
for the args, yes. But not the single text line.